hw/top_englishbreakfast/util/sw_sources.patch

```diff
@@ -70,10 +70,7 @@ index 5dc1286af..1e2dced96 100644
 -  LOG_INFO("Initializing AES unit.");
    init_aes();
 
--  LOG_INFO("Starting simple serial packet handling.");
-   while (true) {
-     simple_serial_process_packet();
-   }
+   LOG_INFO("Starting simple serial packet handling.");
```

Comment:
Can there be unintended changes in this file?

Reply:
Thanks @alphan, good catch! Fixed.